File: includes/undisposable_clients/python/xmlrpc/undisposable.inc.py
#!/usr/bin/python
import xmlrpclib

def isDisposableEmail(email):
    server = xmlrpclib.ServerProxy("http://www.undisposable.org/services/xmlrpc/isDisposableEmail/index.php")
    result = server.isDisposableEmail(email)
    return result

def isDisposableHost(host):
    server = xmlrpclib.ServerProxy("http://www.undisposable.org/services/xmlrpc/isDisposableHost/index.php")
    result = server.isDisposableHost(host)
    return result

File: includes/undisposable_clients/perl/xmlrpc/undisposable.inc.pl
#!/usr/bin/perl
use Frontier::Client;

sub isDisposableEmail {
    my ($email) = @_;
    
    # Make an object to represent the XML-RPC server.
    $server_url = 'http://www.undisposable.org/services/xmlrpc/isDisposableEmail/index.php';
    $server = Frontier::Client->new(url => $server_url);

    # Call the remote server and get our result.
    $result = $server->call('isDisposableEmail',$email);
    return $result;
}

sub isDisposableHost {
    my ($host) = @_;
    
    # Make an object to represent the XML-RPC server.
    $server_url = 'http://www.undisposable.org/services/xmlrpc/isDisposableHost/index.php';
    $server = Frontier::Client->new(url => $server_url);

    # Call the remote server and get our result.
    $result = $server->call('isDisposableHost',$host);
    return $result;
}
